python/cudf/cudf/tests/test_series.py

```diff
@@ -2708,3 +2708,44 @@ def test_series_from_large_string():
     expected = pd.Series(pa_large_string_array)
 
     assert_eq(expected, got)
+
+
+@pytest.mark.parametrize(
+    "scalar",
+    [
+        1,
+        1.0,
+        "a",
+        datetime.datetime(2020, 1, 1),
+        datetime.timedelta(1),
+        {"1": 2},
+        [1],
+        decimal.Decimal("1.0"),
+    ],
+)
+def test_series_to_pandas_arrow_type_nullable_raises(scalar):
+    pa_array = pa.array([scalar, None])
+    ser = cudf.Series(pa_array)
+    with pytest.raises(ValueError):
```

Comment:
For readability, perhaps specify the reason argument `reason="cannot both be set"`?